README.md

```diff
@@ -0,0 +1,26 @@
+## Command Line Interface Calculator
+
+### based on Shunting yard algorithm
+
+#### SDK: OpenJDK 21.0.2
+
+To use program:
```

Comment:
And also provide the step to pack your compiled `.class` files into `.jar` file with appropriate manifest that contains entry point
Then you can run this .jar file directly